ThirdParty/Bolts/BoltsInClasspath/README.md.txt

```diff
@@ -0,0 +1,73 @@
+The sample uses weather.com free subscription rest API to demonstrate the Bolts usage. Please register for free and use the token generated from Weather.com and replace it 
+with apiKey placeholder in the Constants.m file
+
+Note: Make sure this folder and Winobjc release folders exist in the same directory. otherwise it may not run.
```

Comment:
What does this mean? Why does this folder and the release folder need to be in the same directory?